falcon/app.py

```diff
@@ -880,6 +870,9 @@ def _compose_status_response(self, req, resp, http_status):
         # NOTE(kgriffs): If http_status.body is None, that's OK because
         # it's acceptable to set resp.body to None (to indicate no body).
         resp.body = http_status.body
+        # NOTE(caselit): Reset the data and media, so that the body is
+        # sent in all cases, even when it's None
```

Comment:
Could explain the "even when it's `None`" part? How can the body be sent if it is `None`?

Reply:
I'll reword it.
What I meant it that we should not use data or media of body is `None`